### ISSUE TITLE
Check Author-URI and Theme-URI availability

### ISSUE THREAD

**Issue by timelsass** [WPTRT/theme-sniffer#119]
Feature Request:

When doing the sniff on author-uri and theme-uri's it would be nice to check if they also load successfully.  If they are just 404 we could provide authors with informative text that the URLs must be active/relevant.

**Comment by dingo-d** [comment]
Seems doable. Just one thing that's a bit worrisome to me is if we want to use theme sniffer on wordpress.org site (like a stand in for current theme check), how much of an impact to performance would that have?

**Comment by timelsass** [comment]
I don't think it would be very resource intensive done server side.  If it would only be done when a user is uploading a zip, I couldn't imagine it being much of an impact at all.  We could also pass the data parsed to js and do the request clientside

**Comment by dingo-d** [comment]
Was this implemented or does a PR exist for this?
